### DISCUSSION TITLE
New tech report prepared using csasdown

### DISCUSSION THREAD

**Discussion by barbaraneves1** [pbs-assess/csasdowntex#244] [New published reports]
Hello, here's a recent publication prepared using csasdown. Thanks for it!

Neves, B.M., Auger, V., Edinger, E., Lockhart, P., and Morrissey, C. 2023. Preparing for remotely operated vehicle (ROV) seafloor surveys: a descriptive checklist for ocean scientists and managers. Can. Tech. Rep. Fish. Aquat. Sci. 3578: ix + 37 p.

**Comment by seananderson** [comment]
Thanks, added! Do you have a URL?

**Reply by barbaraneves1** [reply]
Here it is: [https://waves-vagues.dfo-mpo.gc.ca/library-bibliotheque/41212459.pdf](url) Thanks!

**Comment by seananderson** [comment]
Thanks! Done.
